data/crac/crac-loopflow-extension/src/main/java/com/farao_community/farao/data/crac_loopflow_extension/CnecLoopFlowExtension.java

```diff
@@ -6,59 +6,73 @@
  */
 package com.farao_community.farao.data.crac_loopflow_extension;
 
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.commons.PhysicalParameter;
+import com.farao_community.farao.commons.Unit;
 import com.farao_community.farao.data.crac_api.Cnec;
 import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.Network;
 
 /**
  * Cnec extension for loop flow
  *
  * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ * @author Baptiste Seguinot {@literal <baptiste.seguinot at rte-france.com>}
  */
 public class CnecLoopFlowExtension extends AbstractExtension<Cnec> {
 
-    private double inputLoopFlow; //input loop flow threshold from TSO for each cross zonal Cnec. absolute value in MW
-    private double loopFlowConstraint; // = max (Abs(inputLoopflow), Abs(initial loopflow))
-    private double loopflowShift; //ptdf * net position
+    // INPUTS
+    private double inputThreshold;
+    private Unit inputThresholdUnit;
+    /*
+     - if the unit is PERCENT_IMAX, the input flow threshold should be between 0 and 100
+     - in the loop-flow threshold, PERCENT_IMAX is considered as a percentage of the Cnec
+       threshold (retrieved from the Crac, without considering the frm), and NOT as a percentage
+        of the branch current limit (retrieved from the Network)
+     */
```

Comment:
I think it's clearer to put the comment above the variable

Reply:
oki